### ISSUE TITLE
Loss is nan on training  RefineDet using LISA traffic dataset 

### ISSUE THREAD

**Issue by StephenGreat** [lzx1413/PytorchSSD#46]
I used LISA traffic dataset as the training dataset and translated all images and annotations to VOC format.
But when training, **the AL, AC and OL are nan**:
Epoch:1 || epochiter: 220/694|| Total iter 220 || AL: 3.7427 AC: 4.2145 OL: 2.7985 OC: 3.4148||Batch time: 0.8865 sec. ||LR: 0.00100000
Epoch:1 || epochiter: 230/694|| Total iter 230 || **AL: nan AC: nan OL: nan** OC: 518.2673||Batch time: 0.7096 sec. ||LR: 0.00100000
Epoch:1 || epochiter: 240/694|| Total iter 240 || **AL: nan AC: nan OL: nan** OC: 0.8749||Batch time: 0.6676 sec. ||LR: 0.00100000

My platform is Pytorch v0.4, cuda9.2.
I have modified the code in refine_multibox_loss.py because the dimensions of tensors(loss_c and pos ) are not matched. Is that reason?

The source code:
#Hard Negative Mining
loss_c[pos] = 0
My code:
loss_c[pos.view(-1,1)] = 0 

Could you help me ?
@lzx1413

**Comment by lzx1413** [comment]
I am working on the version 0.4 now. I will test the performance soon on 0.4. more details can be seen in branch 0.4.
